src/RacingGame.js

```diff
@@ -0,0 +1,46 @@
+const Car = require('./Car');
+const OutputView = require('./UI/OutputView');
+
+class RacingGame {
+  constructor() {}
+
+  setCarList(carNames) {
+    this.carList = carNames.map((carName) => new Car(carName));
+  }
+
+  setAttempts(attempts) {
+    this.attempts = attempts;
+  }
+
+  moveAllCars() {
+    if (this.attempts === 0) return;
+    this.attempts -= 1;
+    this.carList.forEach((car) => car.move());
+  }
+
+  findWinner() {
+    const maxPosition = this.#findMaxPosition();
+    const winners = this.carList.filter((car) => car.position === maxPosition);
+    return winners.map((car) => car.name);
+  }
+
+  #findMaxPosition() {
+    let maxPosition = 0;
+    this.carList.forEach((car) => {
+        if (car.position > maxPosition) maxPosition = car.position;
+    });
+    return maxPosition;
```

Comment:
js native 문법으로 최대값을 찾아보면 어떨까요?

Reply:
```suggestion
   #findMaxPosition() {
    return Math.max(...this.carList.map((car) => car.position));
  }
```
Math 내장함수 max를 활용해 바꾸어보았는데 어떨까요?